src/DssDirectDepositHub.sol

```diff
@@ -298,7 +288,7 @@ contract DssDirectDepositHub {
         if (assetBalance > daiDebt) {
             fees = assetBalance - daiDebt;
```

Comment:
Use `unchecked { ... }` here and in other places where we don't need safemath

Reply:
Can't comment on the line since it's not updated, but in the Hub exec function, can also do:

`toUnwind = Art - lineWad;` as unchecked I think.